Services/Mail/classes/AutoResponder/class.ilAutoResponderServiceImpl.php

```diff
@@ -0,0 +1,117 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ ********************************************************************
+ */
+
+class ilAutoResponderServiceImpl implements ilAutoResponderService
+{
+    protected bool $auto_responder_status;
+    protected array $auto_responder_data;
```

Comment:
Could you please specify the shape of the array by using `PhpDoc` comments, if possible?